src/main/java/org/cftoolsuite/cfapp/notifier/EmailNotifier.java

```diff
@@ -75,17 +75,17 @@ public void onApplicationEvent(EmailNotificationEvent event) {
         String footer =
             String.format("This email was sent from %s on %s",
                     event.getDomain(), DateTimeFormatter.ISO_DATE_TIME.format(LocalDateTime.now()));
-        String subject = event.getSubject();
+        String subject = String.format("%s , Region: %s", event.getSubject(), event.getDomain());
```

Comment:
Isn't this redundant? We display the domain in the footer which is a placeholder section in the body of the email notification.  Why do we also need it in the subject?

Reply:
This can be useful in the case when there are tons of emails sent from the different regions with the same title at the same time. 
The title are all the same to developers and operators, so  the title attached with the region will be helpful for operator and developers to distinguish numerous notification with same title.

Reply:
Or I can take it out from the title, and it the body, It will be helpful that that in the beginning of the body  it is the format of:
Region: east
Org: org name

The reason being that developers often time reply and ask the questions without replying back the spreadsheet attachment, so operator will keeping ask different group of developers to resend the email with spreadsheet attachment. But if the email body embed the region and org in the beginning of the paragraph, it will help for the operator to troubleshoot developers' email questions without asking the spreadsheet attachment.

Reply:
That looks good!